### DISCUSSION TITLE
Human Organs Jewelry Design [Liuwenhao]

### DISCUSSION THREAD

**Discussion by shengyiran30** [realitydeslab/playshop-summer-2024#22] [Proposal]
###  Research Question:
The counter-disciplining of the current societal value system and the questioning of the value of jewelry.

### **Keywords:**
Consumerism, Value Definition, Socio-Economics, Universal Equivalent

### **Artefact:**
 3D Model ， Imag  ，certificate

### **Concept:**

This piece focuses on counter-disciplining the current or potential future jewelry value system's training (brainwashing) of consumer behavior. Jewelry is the medium that carries this concept, used to express a critique and rebellion against the existing social value assessment system. In the post-human era, traditional jewelry gradually loses its value, while the eternal existence of currency within the social structure indicates that the economic and aesthetic value of jewelry has transcended the basic unit of general equivalence. It has become a socially necessary material to satisfy people's class consciousness. Therefore, when the jewelry value system collapses, people turn to using human organs to replace the status of traditional jewelry, beginning to redefine scarcity and value. They use the uniqueness of human tissues and the social identity of the human body as new standards to re-discipline the masses, creating a new consciousness within human society. This reflects people's strong plasticity in society and the top-down prioritization of valuable items within social classes. It guides people to discover and examine the process in which they are unconsciously disciplined by society.

### Visual Reference / Illustrations
![3009bd9d00a701876c704290b916b9f](https://github.com/realitydeslab/playshop-summer-2024/assets/173656838/0e0a3378-6886-443f-82c4-0b23ebcafe3b)

### References:  
[1]  吴莉君,叶荣国.  文化消费主义视域下青年精神生活的异化及其消弭路径  [J/OL].    西安建筑科技大学学报(社会科学版),    1-8[2024-07-03].    http://kns.cnki.net/kcms/detail/61.1330.C.20240618.1805.004.html.  
  [1]  柳丹阳.  网络电商直播中的符号消费主义文化    [J].  喜剧世界(下半月),  2024,    (05):  67-69.  
  [2]  鄢梦瑶,邓琼云.  论文化消费主义对青年红色记忆的消解及其应对    [J].  齐齐哈尔大学学报(哲学社会科学版),  2024,    (03):  77-81+99.  DOI:10.13971/j.cnki.cn23-1435/c.2024.03.016.
  [3]  张龙丽.  文化消费主义的衍生机制、样态演变与纠治路径    [J].  理论导刊,  2024,    (02):  56-61.  
[3]杨嵘均. 网络符号消费阶层比附取向的政治经济学批判 [J]. 东南学术, 2024, (02): 149-161+248. DOI:10.13658/j.cnki.sar.2024.02.009.
[4]沈佩翔. 青年网络符号消费的生成逻辑及其反思 [J]. 东南学术, 2024, (02): 162-170. DOI:10.13658/j.cnki.sar.2024.02.017.Scott R. Klemmer, Michael Thomsen, Ethan Phelps-Goodman, Robert Lee, and James A. Landay. 2002. Where do web sites come from?: capturing and interacting with design history. In Proceedings of the Conference on Human Factors  (CHI ‘02), 1-8. 
http://doi.acm.org/10.1145/503376.503378
武小菲,魏金鑫. 比较视野下行动者网络理论与驯化理论中的“技术——社会”研究 [J]. 传媒论坛, 2024, 7 (08): 44-47.Psy. 2012. Gangnam Style. Video. (15 July 2012.). Retrieved August 22, 2014 from https://www.youtube.com/watch?v=9bZkp7q19f0
reference experiments：
Little Albert Fear Experiment
 Rhesus Monkey Experiment
Milgram Experiment

**Comment by glitch-kraft** [comment]
**Title**
 Human Organs Jewelry Design

**Tagline**
Human Organs based jewellery design for the post-human era where what we consider valuable matter has radically changed.

**Comment by glitch-kraft** [comment]
In the post-human era, traditional jewellery has gradually lost its value. People have turned to using human organs in the design of jewellery, having discovered their incredible value and striking aesthetics.
